spec/ssh_spec.rb

```diff
@@ -0,0 +1,106 @@
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+
+# Copyright 2020-present Facebook
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'logger'
+require 'base64'
+require 'between_meals/util'
+require 'taste_tester/logging'
+require 'taste_tester/exceptions'
+require 'taste_tester/ssh_util'
+require 'taste_tester/ssh'
+require 'taste_tester/config'
```

Comment:
you should change all these to:
```
require_relative '../lib/taste_tester/logging'
...
```

so the tests run against the repo and not installed code.